solidity/dashboard/src/components/Timer.jsx

```diff
@@ -1,8 +1,54 @@
-import React from "react"
+import React, { useEffect, useState } from "react"
 
-const Timer = ({ target }) => {
-  // TODO implement timer
-  return <>{target}</>
+const Timer = ({ days, hours, minutes, seconds }) => {
+  const [remainingDays, setRemainingDays] = useState(days || 0)
```

Comment:
We should sync props to state with `useEffect` eg:
```
useEffect(() => {
  setRemainingHours(props.hours)
 }, [props.hours]
)
```
Otherwise `useState` won’t initialize on props change.